components/smee/production/ip-allow-list.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
We should copy the comment about where the addresses came from here.

Reply:
done